modules/authentication/src/handlers/local.ts

```diff
@@ -43,6 +45,10 @@ export class LocalHandlers implements IAuthenticationStrategy {
           password: ConduitString.Required,
           invitationToken: ConduitString.Optional,
```

Comment:
if the middleware is enabled, you need to add the "captcha token" as an input as well. As an optional input though since it might be disabled for a specific platform.